src/KKS_Fix_MainGameOptimizations/MainGameOptimizations.cs

```diff
@@ -43,6 +44,7 @@ internal void Start()
             UpdateThrottleDynamicBoneUpdatesRange();
             ThrottleDynamicBoneUpdatesRange.SettingChanged += (sender, args) => UpdateThrottleDynamicBoneUpdatesRange();
             ThrottleDynamicBoneUpdatesViewport = Config.Bind(Utilities.ConfigSectionTweaks, "Pause dynamic bones outside camera view", false, new ConfigDescription("Stops dynamic bone physics in roaming mode for characters that are outside of the camera view (e.g. behind or to the side). Improves performance, but can cause the bones to visibly jerk into place as camera rotates to show the character.\nNeeds 'Throttle dynamic bone updates' to be enabled."));
+            AddHeroines = Config.Bind(Utilities.ConfigSectionTweaks, "Number of heroines to be added to each inn", 0, new ConfigDescription("Maximum number of heroines in each inn will be extended", new AcceptableValueRange<int>(0, 500)));
```

Comment:
You can add a ValueChanged event to this setting that immediately removes any heroines above the limit, instead of using the save prefix patch (unless there are other edge cases where the patch is required).

Reply:
No. It could be reduced and then increased again. It should be safer not to finalize it until the save.